checker/src/org/checkerframework/checker/initialization/InitializationVisitor.java

```diff
@@ -268,7 +268,7 @@ public Void visitBlock(BlockTree node, Void p) {
     protected final List<VariableTree> initializedFields;
 
     @Override
-    public Void visitClass(ClassTree node, Void p) {
+    public void processClassTree(ClassTree node) {
         // call the ATF with any node from this class to trigger the org.checkerframework.dataflow
         // analysis.
         atypeFactory.preProcessClassTree(node);
```

Comment:
This is already part of visitClass, so I don't think we still need to call it here.